docs/theorymanual/decay.rst

```diff
@@ -104,18 +182,50 @@ where the coefficients :math:`k_i` are defined as:
 
 .. math::
 
-    k_i = \frac{\gamma}{\lambda_C} \lambda_i c_i
+    k_i = \frac{\gamma}{\lambda_C} \lambda_i \prod_{j\ne i}^C \frac{\lambda_j}{\lambda_j - \lambda_i}
 
 
 **Last Nuclide Stable:**
 
 .. math::
 
-    k_i = -\gamma \prod_{j=1,i\ne j}^{C-1} \frac{\lambda_j}{\lambda_j - \lambda_i}
+    k_{i\ne C} = -\gamma \prod_{j=1,i\ne j}^{C-1} \frac{\lambda_j}{\lambda_j - \lambda_i}
+
+    k_C = \gamma
+
+
+**Last Nuclide Unstable and pth Almost Stable:**
+
+.. math::
+
+    k_{i\ne p} = -\frac{\gamma\lambda_p}{\lambda_C} \prod_{j\n i,p}^C \frac{\lambda_j}{\lambda_j - \lambda_i}
```

Comment:
Specifically here, and below, you have `\n` instead of `\ne`.

Reply:
Ahh thanks

Reply:
Should be fixed now